scripts/spack/configs/blueos_3_ppc64le_ib_p9/spack.yaml

```diff
@@ -0,0 +1,288 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# Axom Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+spack:
+  config:
+    install_tree:
+      root: $spack/..
+      projections:
+        all: '{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+    misc_cache: $spack/../misc_cache
+    test_stage: $spack/../test_stage
+    build_stage::
+    - $spack/../build_stage
+
+  # Regular TPLs do not need views
+  view: false
+
+  compilers:
```

Comment:
You want `::` here. It tells spack to not use any internal compilers and only use the ones listed here.